src/main/kotlin/uk/gov/dluhc/printapi/service/temporarycertificate/CertificatePdfTemplateDetailsFactory.kt

```diff
@@ -14,26 +14,27 @@ class CertificatePdfTemplateDetailsFactory(
     private val s3Client: S3Client,
     @Value("\${temporary-certificate.certificate-pdf.english.path}") private val pdfTemplateEnglish: String,
```

Comment:
Thats a lot of constructor args (not that this PR has really added any more) ... any thoughts on using a constructor binding config class?
I know that config class would have these same args, so it would just be moving constructor args and fields into another class, but wonder if it might be neater?
Just a thought (and like I said, you might have already considered it)

Reply:
Yeah, have considered but haven't tried yet - but will have a go and see if it tidies things up.  (I was wary of not reusing the same object for English and Welsh - especially before having more official templates)

Reply:
Done - isn't spring configuration clever!